src/maps/components/colorbar-plot.tsx

```diff
@@ -102,11 +101,9 @@ const ColorbarPlot = () => {
         const observation = observations[date][position.index] ?? 0;
         if (observation > lastValue && observation <= bin.value) {
           const key = placename(position, placenames);
-          if (index !== ymdDates.length) {
-            const startValue = formatDate(new Date(date));
-            const endValue = formatDate(new Date(yLabels[index + 1]));
-            data.push({x: key, y: [startValue, endValue]});
-          }
+          const startValue = formatDate(toDateUTC(date));
+          const endValue = yLabels[index + 1];
```

Comment:
We've dropped the date formatting for `endValue` here - was that intentional?

Reply:
Yes - yLabels is a different array created from the ymdDates array, and the labels are already formatted.